json-server/routes.json

```diff
@@ -1,3 +1,3 @@
 {
-    "/api/v1/*": "/$1"  
+    "/api/v1/permissions": "/permissions"  
```

Comment:
I don't think you need to make this change. The wildcards in the old version should have routed correctly.